addons/mail/static/tests/activity/activity.test.js

```diff
@@ -542,12 +563,12 @@ test("Activity avatar should have a unique timestamp", async () => {
         res_model: "res.partner",
     });
     await start();
-    const partner = pyEnv["res.partner"].search_read([["id", "=", serverState.partnerId]])[0];
+    const partner = pyEnv["res.partner"].search_read([["id", "=", pyEnv.user.partner_id]])[0];
```

Comment:
not serverState here??

Reply:
Old diff when `serverState` did not exist. `pyEnv.user` was the only solution to get `partner_id`. I'm reverting this LOC for consistency (and simpler code)